### ISSUE TITLE
[BUG] `RuntimeError: protection exception (16777927)`

### ISSUE THREAD

**Issue by kr1tx** [dashingsoft/pyarmor#1331]
```
$ pyarmor gen  --enable-bcc --assert-call Script.py
INFO     Python 3.11.2
INFO     Pyarmor 8.2.8 (pro), 000000, non-profits(TBD)
INFO     Platform windows.x86_64
INFO     search inputs ...
INFO     find script Script.py
INFO     find 1 top resources
INFO     start to generate runtime files
INFO     target platforms {'windows.amd64'}
INFO     write dist\pyarmor_runtime_000000\pyarmor_runtime.pyd
INFO     patch runtime file
INFO     generate runtime files OK
INFO     start to obfuscate scripts
INFO     process resource "Script"
INFO     obfuscating file Script.py
INFO     start to generate bcc code ...
INFO     generate bcc code successfully
INFO     write dist\Script.py
INFO     obfuscate scripts OK

```

**Comment by kr1tx** [comment]
if only --enable-bcc is removed, it works without problems and if only --assert-call is removed also works without problems

**Comment by jondy** [comment]
I'll check it.

**Comment by jondy** [comment]
Reproduced and will be fixed in next release.

**Comment by jondy** [comment]
Fixed in 8.2.9
